### PR TITLE
Converted promises to async/await

### PR DESCRIPTION
**Pull request by berkbaski**

### CODE REVIEW

File: scripts/pages/pgVideoRecorder.ts [hunk 1]
```diff
@@ -92,70 +91,82 @@ export default class PgVideoRecorder extends PgVideoRecorderDesign {
...
+        this.vwRecord.loadFile(this.video);
+
+        this.lblOldSize.text = `Size without SF: ${(this.video.size / (1024 * 1024)).toFixed(2)}MiB`;
+        this.lblNewSize.text = 'Calculating...';
+
+        Multimedia.convertToMp4({
+            videoFile: this.video,
+            outputFileName: `sf-video-${new Date().toISOString().split('.')[0]}`,
+            onCompleted: ({ video: convertedVideo }) => this.convertToMp4OnCompleted(convertedVideo),
+            onFailure: () => this.convertToMp4OnFailure
```
Comment (furkanarabaci) [inline scripts/pages/pgVideoRecorder.ts]: Here won't work. Either do 
`onFailure: this.convertToMp4OnFailure` 
or
`onFailure: () => this.convertToMp4OnFailure()`

File: scripts/pages/pgVideoRecorder.ts [hunk 2]
```diff
@@ -92,70 +91,82 @@ export default class PgVideoRecorder extends PgVideoRecorderDesign {
...
+    }
+    convertToMp4OnFailure() {
+        console.error('An error has occurred when video converting to mp4');
+    }
+    async shareFile(video: File) {
+        try {
+            if (System.OS === System.OSType.ANDROID) {
+                await permission.getPermission({
+                    androidPermission: Application.Android.Permissions.WRITE_EXTERNAL_STORAGE,
+                    permissionText: global.lang.fileAccessError
```
Comment (furkanarabaci) [inline scripts/pages/pgVideoRecorder.ts:153]: Make sure that this language property exists on en.ts

**Review by furkanarabaci — CHANGES_REQUESTED** [review]

File: scripts/pages/pgVideoRecorder.ts (hunk 1 above)
Comment (berkbaski) [reply scripts/pages/pgVideoRecorder.ts]: I fixed it!

File: scripts/pages/pgVideoRecorder.ts (hunk 2 above)
Comment (berkbaski) [reply scripts/pages/pgVideoRecorder.ts:153]: I fixed it!

**Review by furkanarabaci — APPROVED** [review]

**Pull request merged**